### PR TITLE
Drops Welcome Note

### PR DESCRIPTION
**Pull request by jleandroperez**
### Fix
- Drops client-side WelcomeNote creation
- Wires `Accept-Language` header in both, Account Verification and Account Signup remotes

cc @eshurakov 
Thank yoouuu

### Test
1. Fresh install the app
2. Signup for a new account

- [x] Verify the Welcome Note is there!

### Release
These changes do not require release notes.

### CODE REVIEW

File: Simplenote/AccountVerificationRemote.swift [hunk 1]
```diff
@@ -41,6 +41,7 @@ class AccountVerificationRemote {
         var request = URLRequest(url: verificationURL.appendingPathComponent(base64EncodedEmail),
                                  cachePolicy: .reloadIgnoringLocalAndRemoteCacheData,
                                  timeoutInterval: RemoteConstants.timeout)
+        request.setValue(Locale.current.languageCode, forHTTPHeaderField: "Accept-Language")
```
Comment (eshurakov) [inline Simplenote/AccountVerificationRemote.swift]: I don't think you need to manually set it.

To test I commented the code to set `Accept-Language` header and added this snippet to check which headers are sent:
```
        dataTask.resume()
        DispatchQueue.main.async {
            print(dataTask.currentRequest?.allHTTPHeaderFields)
        }
```

And the result is:
```
allHTTPHeaderFields : Optional<Dictionary<String, String>>
      ▿ some : 5 elements
        ▿ 0 : 2 elements
          - key : "Content-Type"
          - value : "application/json"
        ▿ 1 : 2 elements
          - key : "Content-Length"
          - value : "21"
        ▿ 2 : 2 elements
          - key : "Accept-Encoding"
          - value : "gzip, deflate, br"
        ▿ 3 : 2 elements
          - key : "Accept-Language"
          - value : "en-gb"
        ▿ 4 : 2 elements
          - key : "Accept"
          - value : "*/*"

```

**Review by eshurakov — APPROVED** [review]
Works well! 

Sent you a comment about `Accept-Language` :-)

File: Simplenote/AccountVerificationRemote.swift (hunk 1 above)
Comment (jleandroperez) [reply Simplenote/AccountVerificationRemote.swift]: SO MUCH better!! Thank you!!

**Comment by jleandroperez** [comment]
Thank you Evgeny!!

**Comment by pachlava** [comment]
[testing] fix/feature verified on MBP 16" Intel | macOS Big Sur 11.2 | Simplenote 2.9 (11223)

**Pull request merged**